crates/oxc_semantic/src/binder.rs

```diff
@@ -158,6 +158,18 @@ impl<'a> Binder for FormalParameters<'a> {
     }
 }
 
+impl<'a> Binder for TSTypeAliasDeclaration<'a> {
+    fn bind(&self, builder: &mut SemanticBuilder) {
+        // My understanding is this makes type foo = 1; type foo = 2; illegal
```

Comment:
Instead of a comment like this, we need to always reference typescript, and align the symbol flags.

https://github.com/microsoft/TypeScript/blob/71c56c24b6b2939323cb032934c5d9a0d95c413c/src/compiler/binder.ts#L2925-L2926

Reply:
this is what I thought - the comment was more left there because I wasn't exactly sure if my understanding was correct and I wanted confirmation so thank you - since only typeAlias was implemented I put the the exclude as a type alias rather than a more general term - I can do larger clean up and better commenting